pallets/logion_loc/src/lib.rs

```diff
@@ -284,6 +321,59 @@ pub mod pallet {
 				}
 			}
 		}
+
+		/// Make a LOC void.
+		#[pallet::weight(T::WeightInfo::make_void())]
+		pub fn make_void(
+			origin: OriginFor<T>,
+			#[pallet::compact] loc_id: T::LocId,
+			reason: Vec<u8>,
+			replacer_loc_id: Option<T::LocId>
```

Comment:
From a Polkadot JS perspective, I think that it would be better after all to have 2 methods: one taking a replacer LOC ID and another which does not. The reason is that the representation of the loc_id and replacer_loc_id parameters would be different with the single method option (one is a compact, the other not so we have to fallback on some hex representation) where the two methods option enables to represent both IDs in the form of compacts.